packages/aws-cdk/test/api/deploy-stack.test.ts

```diff
@@ -61,15 +61,17 @@ test('correctly passes CFN parameters, ignoring ones with empty values', async (
     resolvedEnvironment: mockResolvedEnvironment(),
     parameters: {
       A: 'A-value',
-      B: undefined,
-      C: '',
+      B: 'B=value',
+      C: undefined,
+      D: '',
     },
   });
 
   // THEN
   expect(cfnMocks.createChangeSet).toHaveBeenCalledWith(expect.objectContaining({
     Parameters: [
       { ParameterKey: 'A', ParameterValue: 'A-value' },
+      { ParameterKey: 'B', ParameterValue: 'B=value' },
```

Comment:
we usually try not to modify existing tests so we don't end up masking a regression. I think in this scenario it's okay though since it's a superset of what we used to test.

Reply:
yeah, i've read that in the documentation, but figured that this shouldn't be a breaking change

Reply:
yeah, i've read that in the documentation, but figured that this shouldn't be a breaking change

Reply:
I agree, it doesn't lose any of the initial test case. let's leave this as it is.